### PR TITLE
DOCS-625: Removing FS Mode guidance, cleanups to deploy docs

### LINKED ISSUE #625: Revise Deployment Docs for deprecated/removed FS mode
**Summary**

We've fully removed FS mode from MinIO, and our docs need a bit of clean-up and tidying in general around deployment topologies

**Goals**

List the in-scope goals

- [ ] Clean up the deployment comparison page
- [ ] Add clear warnings about deprecation of the FS/Gateway
- [ ] Light touch-ups and consistency across the deployment topologies

**Non-Goals**

- Nice graphical descriptions of our supported topologies
- Migration guide (to be handled in #624

### PR DESCRIPTION
**Pull request by ravindk89**
Closes #625

### CODE REVIEW

File: source/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.rst [hunk 1]
```diff
@@ -11,11 +11,11 @@ Deploy MinIO: Single-Node Multi-Drive
    :depth: 2
 
 The procedures on this page cover deploying MinIO in a Single-Node Multi-Drive (SNMD) configuration.
-This topology provides increased drive-level reliability and failover protection as compared to :ref:`Single-Node Single-Drive (SNSD) deployments <minio-snsd>`.
+|MNMD| deployments provide drive-level reliability and failover/recovery with node-level performance and scaling constraints.
```
Comment (djwfyi) [inline source/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.rst:14]: Should this be `|SNMD|`?

Comment (djwfyi) [reply source/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.rst:14]: Maybe ...
> ... and failover/recovery with performance and scaling limitations imposed by the single node.

File: source/operations/install-deploy-manage/deploy-minio-single-node-single-drive.rst [hunk 2]
```diff
@@ -27,6 +27,13 @@ See the documentation on :ref:`SNSD behavior with pre-existing data <minio-snsd-
 
    For extended development or production environments, deploy MinIO in a :ref:`Multi-Node Multi-Drive (Distributed) <minio-mnmd>` topology
 
+.. important::
+
+   :minio-release:`RELEASE.2022-10-29T06-21-33Z` fully removes the deprecated Gateway/Filesystem backends.
```
Comment (djwfyi) [inline source/operations/install-deploy-manage/deploy-minio-single-node-single-drive.rst:32]: maybe link `deprecated Gateway/Filesystem backends` to the blog article.

**Review by djwfyi — APPROVED** [review]
Couple of fixes and a wording suggestion for better clarity, otherwise LGTM.

Fix it up and merge it.

File: source/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.rst (hunk 1 above)
Comment (ravindk89) [reply source/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.rst:14]: That's a good rewrite

**Pull request merged**